src/build.d

```diff
@@ -163,30 +163,37 @@ auto lexer()
 /// Returns: the dependency that generates the dmd.conf file in the output folder
 auto dmdConf()
 {
-    // TODO: add support for Windows
-    string exportDynamic;
-    version(OSX) {} else
-        exportDynamic = " -L--export-dynamic";
-
-    auto conf = `[Environment32]
-DFLAGS=-I%@P%/../../../../../druntime/import -I%@P%/../../../../../phobos -L-L%@P%/../../../../../phobos/generated/{OS}/{BUILD}/32{exportDynamic}
-
-[Environment64]
-DFLAGS=-I%@P%/../../../../../druntime/import -I%@P%/../../../../../phobos -L-L%@P%/../../../../../phobos/generated/{OS}/{BUILD}/64{exportDynamic} -fPIC`
-        .replace("{exportDynamic}", exportDynamic)
-        .replace("{BUILD}", env["BUILD"])
-        .replace("{OS}", env["OS"]);
+    const makedmdconfBin = env["G"].buildPath("makedmdconf").exeName;
+    Dependency buildMakedmdconf = {
```

Comment:
```suggestion
    auto buildMakedmdconf = {
```

This will make it easier if we ever change Dependency to e.g. DependencyRef or similar.